src/components/Alert.vue

```diff
@@ -1,122 +0,0 @@
-<template>
```

Comment:
Spite we need to delete this component, I think it's out the scope of this big PR